tasks/libs/ciproviders/gitlab_api.py

```diff
@@ -1185,3 +1196,35 @@ def compute_gitlab_ci_config_diff(ctx, before: str, after: str):
     diff = MultiGitlabCIDiff.from_contents(before_config, after_config)
 
     return before_config, after_config, diff
+
+
+def get_special_jobs(lint=False, all_jobs=None, all_stages=None):
+    """
+    Parses the special jobs file and lints it.
+
+    - lint: If True, will lint the file to verify that each job / stage is present in the configuration
+    - all_jobs: All the jobs in the configuration
```

Comment:
Maybe you could add some more details about what is expected for this variable? This is not very clear what should be put there without reading the code